app/include/dt-bindings/zmk/backlight.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define BL_TOG 0
```

Comment:
Would it be useful to have an option to set the backlight to a specific level with a key? (If so, you'd need to do something like the `&bt` where the behavior takes two params, and the macros are things like

```c
#define BL_DEC 4 0
#define BL_SET 5
```
```c

&bl BL_SET 100
```

Reply:
I don't find it particularly useful, but other opinions would be helpful